src/MICore/LaunchOptions.cs

```diff
@@ -64,6 +64,14 @@ public enum LaunchCompleteCommand
     /// </summary>
     public sealed class PipeLaunchOptions : LaunchOptions
     {
+        /// <summary>
+        /// Creates an instance of PipeLaunchOptions
+        /// </summary>
+        /// <param name="pipePath">Path of the pipe program</param>
```

Comment:
Might want to add something here, and to the xsd file (line 292) something like --

This should either be a full path to an executable or the name of a program on the PATH.